src/pages/wallet/send.vue

```diff
@@ -282,18 +317,34 @@ export default {
       this.newTx.address = info.address;
       this.newTx.payment_id = info.payment_id;
     },
-    // helper for constructing a dialog for confirming transactions
-    confirmTransactionDialog(val) {
-      // the split_transfer rpc call can return several txs
+    buildDialogFields(val) {
+      this.confirmTransaction = true;
```

Comment:
Is this ever set to `false` after we set it to `true`? What happens when you send transactions in a row, will the dialog show again?

Reply:
`v-close-popup` on the dialog buttons sets this to false.